json-logs/samples/events/AppMentionPayload.json

```diff
@@ -1,35 +1,35 @@
 {
   "token": "",
-  "team_id": "T00000000",
-  "enterprise_id": "E00000000",
-  "api_app_id": "A00000000",
+  "team_id": "",
```

Comment:
This used to be relying on my local assets. Now everyone can generate the same sample JSON files if they properly set up test workspaces.